testing/client_app/pages/login/components/log_in_component.py

```diff
@@ -0,0 +1,48 @@
+from framework.abstract.component_base import BaseComponent
+from framework.factory.models_enums.constants_enums import LocatorType
+from selenium import webdriver
+from framework.factory.models_enums.page_config import ComponentModel
+
+
+class LogInComponent(BaseComponent):
+    """
+    This class is a component of the log-in page.
+        It handles all events and UI locators of log-in part.
+    """
+    def __init__(self, driver: webdriver, component_data: ComponentModel):
```

Comment:
Constructors should return `None`.

```suggestion
    def __init__(self, driver: webdriver, component_data: ComponentModel) -> None:
```